test/OWNERS

```diff
@@ -8,6 +8,7 @@ reviewers:
 approvers:
   - alejandrox1
   - andrewsykim
+  - BenTheElder
```

Comment:
Though it looks odd to add yourself to a cleanup PR. guess that's my review on the way getting kicked out  *smile*

Reply:
I agree, except that I'm not actually adding myself. I'm removing myself from being added via an alias (as most of the other entries in the team / alias are already explicit).

I've been in `test/` approvers here via the sig-testing-approvers team / alias for O(years) and i'm quite active here.

The subtext is that I'd like to remove the concept of """sig-testing-approvers""" owning various directories instead of specific people, but to do that I needed to own it without the alias. I didn't quite finish that part, I'll get to it in a future cleanup.

Reply:
IMHO it's a huge anti-pattern to opaquely grant people ownership of multiple directories via these aliases, and now that I chair SIG Testing I intend to have SIG Testing stop doing this. We don't have many people in the OWNERS_ALIASES entry anyhow.